multi_physics/QED/QED_tests/test_picsar_algo.cpp

```diff
@@ -11,8 +11,8 @@
 #include <algorithm>
```

Comment:
usually include stdlib includes last, so you notice quickly when an include is missing in a header file

Reply:
I have just rearranged the stdlib includes in all the tests!